app/run-test.sh

```diff
@@ -26,9 +26,11 @@ fi
 testcase="$path"
 echo "Running $testcase:"
 
-west build -d build/$testcase -b native_posix_64 -- -DZMK_CONFIG="$(pwd)/$testcase" > /dev/null 2>&1
+mkdir -p build/$testcase
+west build -d build/$testcase -b native_posix_64 -- -DZMK_CONFIG="$(pwd)/$testcase" >build/$testcase/build.log 2>&1
 if [ $? -gt 0 ]; then
     echo "FAILED: $testcase did not build" | tee -a ./build/tests/pass-fail.log
+    echo $'\t' "see ./build/$testcase/build.log"
```

Comment:
```suggestion
    echo '\t'"see ./build/$testcase/build.log"
```
`$'..'` seems to be a bash-ism, doesn't work with `dash` in Ubuntu here.

Reply:
Can't say I write much crossplatform shell script :smiley: . Your version prints `\t` in bash. How about

```suggestion
    printf '\t%s' "see ./build/$testcase/build.log"
```

Reply:
That's probably the way to go, yeah.

Reply:
Thanks for updating! To give some context to my initial comment: the script has shebang `#!/bin/sh`, which is only guaranteed to be a POSIX-compatible shell. That is *usually* not bash, for instance in Debian-based systems it will be [dash](https://en.wikipedia.org/wiki/Almquist_shell#dash). Not all POSIX-compatible shells support the same features, but they are all supposed to support the POSIX spec. So `#!/bin/sh` scripts should avoid bash-isms (or dash-isms, as it turns out). (https://github.com/dylanaraps/pure-sh-bible is a great resource if you are interested)